mmv1/products/clouddeploy/DeployPolicy.yaml

```diff
@@ -0,0 +1,319 @@
+    send_empty_value: true
+  - name: "selectors"
+    type: Array
+    description: "Required. Selected resources to which the policy will be applied. At least one selector is required. If one selector matches the resource the policy applies. For example, if there are two selectors and the action being attempted matches one of them, the policy will apply to that action."
+    required: true
+    item_type:
+      type: NestedObject
+      properties:
+        - name: "target"
+          type: NestedObject
+          description: "Contains attributes about a target."
+          properties:
+            - name: "id"
+              type: String
+              description: 'ID of the `Target`. The value of this field could be one of the following: * The last segment of a target name. It only needs the ID to determine which target is being referred to * "*", all targets in a location.'
+            - name: "labels"
+              type: KeyValuePairs
+              description: "Target labels."
+              default_from_api: true
+        - name: "deliveryPipeline"
+          type: NestedObject
+          description: "Contains attributes about a delivery pipeline."
+          properties:
+            - name: "id"
+              type: String
+              description: |-
+                Optional. ID of the DeliveryPipeline. The value of this field could be one of the following:
+                - The last segment of a pipeline name
+                - "*", all delivery pipelines in a location
+            - name: "labels"
+              type: KeyValuePairs
+              description: "DeliveryPipeline labels."
+              default_from_api: true
+  - name: "rules"
+    type: Array
+    description: "Required. Rules to apply. At least one rule must be present."
+    required: true
+    item_type:
+      type: NestedObject
+      properties:
+        - name: "rolloutRestriction"
+          type: NestedObject
+          description: "Optional. Rollout restrictions."
+          properties:
+            - name: "id"
+              type: String
+              description: "Required. ID of the rule. This id must be unique in the `DeployPolicy` resource to which this rule belongs. The format is `a-z{0,62}`."
+              required: true
+            - name: "invokers"
+              type: Array
+              description: "Optional. What invoked the action. If left empty, all invoker types will be restricted."
+              item_type:
+                type: Enum
+                enum_values:
+                  - "INVOKER_UNSPECIFIED"
+                  - "USER"
+                  - "DEPLOY_AUTOMATION"
+            - name: "actions"
+              type: Array
+              description: "Optional. Rollout actions to be restricted as part of the policy. If left empty, all actions will be restricted."
+              item_type:
+                type: Enum
+                enum_values:
+                  - "ROLLOUT_ACTIONS_UNSPECIFIED"
```

Comment:
_UNSPECIFIED is probably an invalid option to specify. I would remove this

Reply:
Done